Naive greedy alg/scheduling/__init__.py

```diff
@@ -1,14 +1,63 @@
 import logging
+import pandas as pd
 
-from .Swarm import Swarm
+# from .Swarm import Swarm
+from .Experiments import Experiments
+from .Scheduler import SchedulerConfig
+
+# python -m scheduling
 
 
 def main():
     logging.basicConfig(
         level=logging.INFO, format="%(asctime)s %(name)s %(levelname)-5s: %(message)s"
     )
 
-    swarm = Swarm(seed_num=1, num_particles=10, num_srvs=10, num_exp=10)
-    # swarm.configs()
-    swarm.run_epochs(num_epochs=1, draw_stats=True)
-    # swarm.configs()
+    # swarm = Swarm(seed_num=1, num_particles=30, num_srvs=10, num_exp=50)
+    # # swarm.configs()
+    # swarm.run_epochs(num_epochs=100, draw_stats=False, draw_graph=True)
+    # # swarm.configs()
+
+    n_servers = 10
+    n_expts = 30
+    seed = 2
+    # run expt with no reconfig and no power off
```

Comment:
I suggest we do 1 function that takes the csv name and **kwargs and then unpack the dictionary when calling run_exps, like run_expts(**kwargs)

Reply:
Seems fine to me. You can implement it if you have time